internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -38,7 +38,7 @@ struct {
 } events SEC(".maps");
 
 struct event_t {
-    u64 pid;
+    u32 pid;
```

Comment:
IIRC `pid_t` is a 32bit type so this change is correct, but I wonder (just for my curiosity, really), where does `u32` come from? It took me a bit to realize I'm not looking at `uint32_t` and I was wondering why we were not using `pid_t` in the first place here.

Is the type a BPF type?

Reply:
Yes, `vmlinux.h` provides typedefs for them: https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/internal/pkg/daemon/bpfrecorder/vmlinux/x86/6.0.7-301.fc37.x86_64.vmlinux.h